osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -62,6 +64,8 @@ public abstract class SongSelect : OsuScreen
         private SampleChannel sampleChangeDifficulty;
         private SampleChannel sampleChangeBeatmap;
 
+        protected new readonly Bindable<RulesetInfo> Ruleset = new Bindable<RulesetInfo>();
```

Comment:
Are you sure we don't want to be re-DI'ing this ruleset in and replacing the existing one? Seems like components underneath songselect (leaderboard, wedge) should use this one, no?

Reply:
ah yeah, that got lost in some refactoring. that was the whole point of this PR